jsonschema-validation.xml

```diff
@@ -649,12 +635,12 @@
 
             <section title="oneOf">
                 <t>
-                    This keyword's value MUST be an array. This array MUST have at least one
-                    element.
+                    This keyword's value MUST be an array.
+                    This array MUST have at least one element.
                 </t>
                 <t>
-                    Elements of the array MUST be objects. Each object MUST be a valid JSON
-                    Schema.
+                    Elements of the array MUST be objects.
```

Comment:
I think maybe I changed it like that in a PR that I ended up dropping?  The boolean schema thing went through a few variations before we settled on it.  Whatever happened, I think just saying "valid JSON Schema" is the best option- we already define what that means in an obvious place (the core spec).